### PR TITLE
Issue-403

### PR DESCRIPTION
**Pull request by cmoulliard**
- See Issue: #403 
- WIP

### CODE REVIEW

File: app/src/test/java/io/halkyon/CredentialsPageTest.java [hunk 1]
```diff
@@ -15,9 +15,12 @@
 public class CredentialsPageTest extends BaseTest {
 
     @Test
-    public void testCreateNewCredential() {
+    public void testCreateNewBasicCredential() {
```
Comment (aureamunoz) [inline app/src/test/java/io/halkyon/CredentialsPageTest.java:18]: Rename to testCreateUserPasswordCredential? it's more meaningful

**Review by aureamunoz — CHANGES_REQUESTED** [review]
Other than a minor renaming propal LGTM

File: app/src/test/java/io/halkyon/CredentialsPageTest.java (hunk 1 above)
Comment (cmoulliard) [reply app/src/test/java/io/halkyon/CredentialsPageTest.java]: Fixed with: cfc1f25

**Review by aureamunoz — APPROVED** [review]
LGTM

**Review by aureamunoz — APPROVED** [review]

**Pull request merged**